overlord/ifacestate/implicit.go

```diff
@@ -42,15 +42,16 @@ func addImplicitSlots(snapInfo *snap.Info) {
 		if (release.OnClassic && md.ImplicitOnClassic) || (!release.OnClassic && md.ImplicitOnCore) {
 			ifaceName := iface.Name()
 			if _, ok := snapInfo.Slots[ifaceName]; !ok {
-				snapInfo.Slots[ifaceName] = makeImplicitSlot(snapInfo, ifaceName)
+				snapInfo.Slots[ifaceName] = makeImplicitSlot(snapInfo, ifaceName, md.Summary)
```

Comment:
I think the label is not a super useful way to populate the implicit interfaces. We could come up with a better one (specifically tailored to the implicit interface creation). In other news the way this information is exposed over the REST API is bound to change with the #3399 PR (which needs to be redone but now contains the description of the indented design)